### PR TITLE
Add omega watches 

### LINKED ISSUE #3: Add Watch Models for Various Brands
**Description**:
- We are currently working on expanding our collection of watch models, and we need your help! Your task is to add watch models based on the brands listed in the project. The data should follow the specific format found in the `script.js` file under the `const watches = [` section.

**Instructions**:
- Images such as the "_**rolex folder**_" should be added to the main code.
- Select a watch brand from the list provided below.
- Add a minimum of 5 and a maximum of 20 watch models for the selected brand.

**Make sure the data follows the format below**:

```
{
    name: "Breitling Navitimer",
    dialColor: "black",
    strapColor: "brown",
    techLevel: "analog",
    img: "breitling/Breitling-Navitimer.png",
    url: "https://www.breitling.com/us-en/watches/navitimer/"
}
```
**Watch Brands**:
- Breitling
- Rolex (_Currently included as an example in the main file but can be increased_.)
- Omega
- Tag Heuer
- Seiko

**_Additional Information_**:
- Please ensure that the img field follows the naming convention: brand-name/watch-model.png.
- **Provide a valid URL for the watch in the url field**. (Must definitely match.)
- Submit a pull request once your changes are complete.

_Good luck!_

### PR DESCRIPTION
**Pull request by SeanAverS**
Fixes #3 

## Changes 
- Create `omega` folder to main code and add 5 images to it
- Add omega watch model data in `script.js` for `omega` images

### CODE REVIEW

**Comment by brgkdm** [comment]
> Fixes #3
> 
> ## Changes
> * Create `omega` folder to main code and add 5 images to it
> * Add omega watch model data in `script.js` for `omega` images

Dear Contributor,

Thank you very much for your recent pull request and for the effort you have put into improving the project. I have reviewed the changes you proposed, and overall, the work looks quite promising. However, I have noticed an issue with the images you added in the `omega` folder. Specifically, one of the images is not displaying correctly due to a punctuation error in the code. This appears to be a minor issue, likely related to incorrect syntax in your `script.js` file, which is preventing the image from being rendered as expected.

To resolve this, I kindly ask you to review the code again, particularly focusing on the section where you reference the image files in `script.js`. It’s possible that there is either a missing or misplaced character, such as a comma or quotation mark, which could be causing this error. Once you have corrected it, I would appreciate it if you could test the changes locally to ensure all images display properly.

Afterward, please feel free to resubmit the pull request with the necessary corrections. I will be happy to review it again and proceed accordingly.

Thank you for your attention to this matter, and I look forward to your updated submission.

Best regards,  
[Berat Gökdemir]

invalidwatchcode1

invalidpicwatch

invalidwatchcode2

truepicwatch

File: WatchLore/script.js [hunk 1]
```diff
@@ -73,6 +73,46 @@ const watches = [
...
+        techLevel: "analog",
+        img: "omega/constellation.png",
+        url: "https://www.omegawatches.com/watches/constellation"
+    },
+    {
+        name: "Omega De Ville Ladymatic",
+        dialColor: "white",
+        strapColor: "silver",
+        techLevel: "analog",
+        img: "omega/deville-ladymatic-png",
```
Comment (brgkdm) [inline WatchLore/script.js:113]: > `"omega/deville-ladymatic-png"` (old)

`omega/deville-ladymatic.png` (new)

**Comment by SeanAverS** [comment]
@brgkdm thank you for your patience and detailed instructions 🙏

**Comment by brgkdm** [comment]
> @brgkdm thank you for your patience and detailed instructions 🙏 

Added.

**Pull request merged**